tests/integration/test_data/gomod_packages.yaml

```diff
@@ -0,0 +1,402 @@
+with_deps:
```

Comment:
IMHO, maintaining these output files is very time consuming. For every new Go version, for instance, chances are that they changed some of the stdlibs and we'll need to change a lot stuff in a lot of files.

Any chances we could have a generator function for those? This file is mostly compromised of the Cachi2 request input, the output.json (converted to yaml) and those zip files. The first two are very easy to have, not sure about the last one. 

For that, we'd need to have separate test_data for each test case, and we'd manually run the generator function, skim through the diffs, and commit the changes. 

Wdyt?

Reply:
I agree that we cannot spend time on generating these expected output data manually. I am adding dummy proposal into design. Let's have discussion in there.